src/main/java/com/formulasearchengine/mathmlconverters/MathMLConverter.java

```diff
@@ -194,11 +205,18 @@ private void removeAttribute(Node node, String name) {
      */
     Content scanFormulaNode(Element formulaNode) throws Exception {
         // first off, try scanning for mathml nodes directly
+        Element annotationNode = (Element) XMLHelper.getElementB(formulaNode, xPath.compile("//m:annotation-xml"));
+        Boolean encodingMathML;
+        if (annotationNode != null) {
+            encodingMathML = annotationNode.getAttribute("encoding").equals("MathML-Content");
+        } else {
+            encodingMathML = false;
+        }
         Element semanticNode = (Element) XMLHelper.getElementB(formulaNode, xPath.compile("//m:semantics"));
         NonWhitespaceNodeList applyNodes = new NonWhitespaceNodeList(XMLHelper.getElementsB(formulaNode, xPath.compile("//m:apply")));
         NonWhitespaceNodeList mrowNodes = new NonWhitespaceNodeList(XMLHelper.getElementsB(formulaNode, xPath.compile("//m:mrow")));
         // both variants are present, if the semantics separator is present everything is fine
-        if (applyNodes.getLength() > 0 && mrowNodes.getLength() > 0) {
+        if (applyNodes.getLength() > 0 && mrowNodes.getLength() > 0 || encodingMathML) {
```

Comment:
An annotation node with MathML-Content attribute just means you can be sure to have Content MathML ( which would be Content.cmml).